tensorflow/python/keras/layers/local_test.py

```diff
@@ -288,8 +288,10 @@ def test_locallyconnected_implementation(self, width, data_format):
 
       np.random.seed(1)
       tf_test_util.random_seed.set_seed(1)
-      targets = np.random.randint(0, num_classes, (num_samples,))
-
+      sparse_targets = np.random.randint(0, num_classes, (num_samples,)) 
```

Comment:
Maybe add comments for unintuitive code blocks. These 3 lines create a matrix with each row a one-hot vector, but it's not that obvious.

Reply:
Added comment and made requested changes